src/components/Homepage/HomePage.jsx

```diff
@@ -1,61 +1,105 @@
 import React, { useEffect } from "react";
 import { useState } from "react";
-import { Link } from "react-router-dom";
-const axios = require("axios");
+import { Link, useNavigate, useParams } from "react-router-dom";
+import axios from "axios";
+import Spinner from "../Spinner/Spinner";
 
 const api_key = process.env.REACT_APP_API_KEY;
 
-export default function HomePage({ setMovieInfo }) {
+export default function HomePage({ isLoading, setIsLoading }) {
```

Comment:
isLoading and setIsLoading should be private to that component

Reply:
fixed